c2corg_ui/static/js/user.js

```diff
@@ -100,4 +100,31 @@ app.UserController.prototype.logout = function() {
 };
 
 
+/**
+ * @private
+ */
+app.UserController.prototype.getUserId_ = function() {
+  if (window.localStorage.getItem('userData')) {
```

Comment:
You must retrieve the userData from the auth service, not from the local storage.